build.gradle

```diff
@@ -259,8 +261,10 @@ dependencies {
   integrationTestImplementation group: 'org.testcontainers', name: 'postgresql', version: '1.12.4'
 
   functionalTestImplementation sourceSets.main.runtimeClasspath
+  functionalTestImplementation sourceSets.test.runtimeClasspath
```

Comment:
the purpose of **not** including test sourcesets was that fun and smoke tests are testing running application so no need to know anything in current spring boot manipulations in unit test suite. is there anything specific we use from there in this pr?

Reply:
There was some common stuff. I might get that separately.

Reply:
done